package.json

```diff
@@ -2,7 +2,7 @@
   "name": "flow-playground",
   "private": true,
   "scripts": {
-    "start": "webpack-dev-server -d --config webpack.config.js --inline ",
+    "start": "webpack-dev-server -d source-map --mode=development --config webpack.config.js",
```

Comment:
do you mind explaining how development mode works since i didnt see the mode in the webpack config

Reply:
The latest webpack throws an error without an explicit mode. The mode can be set here, or we can add a default mode to webpack.config.js. Let me know if you prefer that

Reply:
yeah lets add a default mode for readability